frontend/src/components/samples/details/SampleDetailsContent.js

```diff
@@ -352,7 +352,7 @@ const SampleDetailsContent = ({
           </Descriptions>
         </TabPane>
 
-        <TabPane tab={`Lineage`} key="lineage" style={tabStyle}>
+        <TabPane tab={`Lineage`} key="lineage" style={{...tabStyle, minHeight: "100vh", maxheight: "100vh"}}>
```

Comment:
Setting the tab height to 100vh is setting the height to the window height, so the tab bottom extends beyond the bottom of the window, making it scrollable.